common/country_formation/00_major_formables.txt

```diff
@@ -47,9 +47,11 @@ ITA = {
 			country_has_primary_culture = cu:south_italian
 		}
 		any_country = {
-			OR = {
+			filter = {
+				OR = {
 					country_has_primary_culture = cu:north_italian
 					country_has_primary_culture = cu:south_italian				
+				}
 			}
 			has_technology_researched = pan-nationalism
 			percent = 0.5
```

Comment:
This should be `percent >= 0.5` from what I saw in that screenshot, yes?

Reply:
and thats why I asked for a second opinion lol